great_expectations/rule_based_profiler/parameter_builder/metric_multi_batch_parameter_builder.py

```diff
@@ -140,8 +140,12 @@ def _build_parameters(
         )
 
         # As a simplification, apply reduction to scalar in case of one-dimensional metric (for convenience).
-        if reduce_scalar_metric and metric_values.shape[1] == 1:
-            metric_values = metric_values[:, 0]
+        if (
+            reduce_scalar_metric
+            and len(metric_values) == 1
```

Comment:
👍🏼

Reply:
@donaldheppner These types are placeholder ones, which led to needing this logic for certain `ParameterBuilder` implementations.  It was intended to be replaced as part of a MAINTENANCE task (I just created a JIRA item for it).  Thanks.